tokens/build.mjs

```diff
@@ -0,0 +1,84 @@
+import StyleDictionaryPackage from 'style-dictionary'
+const { fileHeader, formattedVariables } = StyleDictionaryPackage.formatHelpers
+
+const themes = ['Reapit', 'PayProp']
+
+const getStyleDictionaryConfig = (theme) => {
+  const lowerCasedTheme = theme.toLowerCase()
+  return {
+    source: ['./tokens/tokens.json'],
+    format: {
```

Comment:
**suggestion:** It seems like we could we simplify this and just use the built-in CSS format to get a CSS file, e.g. tokens.css, then import that file using a CSS `@import` statement in `globals.ts` 🤔

**note:** It's quite possible I'm missing something with Linaria that prevents this from working; though it seems @import statements are permitted in the global.ts as we're using it for the Source Code Pro font.

Reply:
Nice suggestion, yes, this works - have implemented and was able to get rid of the linaria parse function.